usaspending_api/api_docs/urls.py

```diff
@@ -9,6 +9,10 @@
     url(r"^data-dictionary", MarkdownView.as_view(markdown="data_dictionary.md")),
     url(r"^recipes", MarkdownView.as_view(markdown="request_recipes.md")),
     url(r"^using-the-api", MarkdownView.as_view(markdown="using_the_api.md")),
-    url(r"^entity-relationships", Plate.as_view(plate_template_name="plate.html"), name="plate"),
+    url(
```

Comment:
Was this intentional? I don't see why this was changed

Reply:
Not sure what happened here. Seems like it might have been reformatted with black. I changed it back, and black didn't reformat it